server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/AbstractEndpoint.java

```diff
@@ -344,12 +341,19 @@ public void before(final Request the_request, final Response the_response) {
   } 
   
   /**
-   * The after filter for this endpoint. By default, it attempts to commit 
+   * The after filter for this endpoint. Currently, the implementation is empty.
+   */
+  public void after(final Request the_request, final Response the_response) {
+    // skip
+  }
+  
+  /**
+   * The afterAfter filter for this endpoint. By default, it attempts to commit 
    * any open transaction (this makes writing endpoint code more straightforward,
    * as the vast majority of endpoints will never have to deal with transactions
    * themselves).
    */
-  public void after(final Request the_request, final Response the_response) {
+  public void afterAfter(final Request the_request, final Response the_response) {
```

Comment:
Not a fan of this name. Can we call it anything more specific to what it is meant for?